services/dynamodb/src/main/java/software/amazon/awssdk/services/dynamodb/model/ItemBuilder.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright 2010-2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package software.amazon.awssdk.services.dynamodb.model;
+
+import static java.util.stream.Collectors.toList;
+
+import java.nio.ByteBuffer;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import software.amazon.awssdk.services.dynamodb.utils.AttributeValueHelper;
+import software.amazon.awssdk.utils.builder.SdkBuilder;
+
+public final class ItemBuilder implements SdkBuilder<ItemBuilder, Map<String, AttributeValue>> {
+
+    private Map<String, AttributeValue> item = new HashMap<>();
+
+    private ItemBuilder() {}
+
+    public static ItemBuilder builder() {
+        return new ItemBuilder();
+    }
+
+    @Override
+    public Map<String, AttributeValue> build() {
+        return item;
+    }
+
+    public ItemBuilder attr(String key, String stringValue) {
+        item.put(key, AttributeValue.builder().s(stringValue).build());
+        return this;
+    }
+
+    public ItemBuilder attr(String key, Boolean booleanValue) {
+        item.put(key, AttributeValue.builder().bool(booleanValue).build());
+        return this;
+    }
+
+    public ItemBuilder attr(String key, Number numericValue) {
+        item.put(key, AttributeValue.builder().n(String.valueOf(numericValue)).build());
+        return this;
+    }
+
+    public ItemBuilder attr(String key, byte[] binaryValue) {
+        return attr(key, ByteBuffer.wrap(binaryValue));
+    }
+
+    public ItemBuilder attr(String key, ByteBuffer binaryValue) {
+        item.put(key, AttributeValue.builder().b(binaryValue).build());
+        return this;
+    }
+
+    public ItemBuilder strings(String key, String...stringValues) {
```

Comment:
space after ...? Is checkstyle not catching that?

Reply:
Didn't appear to.